portfolio/src/main/webapp/script.js

```diff
@@ -27,6 +27,17 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
+function AddARandomMeme(){
+    
+  fetch('/data?aRandomOne=true').then(response => response.json()).then((resJson) => {
+    document.getElementById('aRandomMeme-container').innerHTML = 
+    '<p> ['+ resJson.comment +'] </p>\n' +
+    '<img src="'+ resJson.url +'" ></img>\n';
+  });
```

Comment:
You could achieve XSS safe without restring the url/comment input:
```suggestion
container = document.getElementById('aRandomMeme-container');
// Empties the container.
container.innerHTML = '';

commentElement = document.createTextNode(resJson.comment);

imgElement = document.createElement('img');
imgElement.src = resJson.url;

container.appendChild(commentElement);
container.appendChild(imgElement);
```